Frontend/ui-customer/src/Flow.purs

```diff
@@ -1972,25 +1993,84 @@ checkAndUpdateSavedLocations state = do
 addLocationToRecents :: LocationListItemState -> HomeScreenState -> Boolean -> Boolean -> FlowBT String Unit
 addLocationToRecents item state srcServiceable destServiceable = do
   let serviceable = if (state.props.isSource == Just false) then destServiceable else srcServiceable
+      lat = if (state.props.isSource == Just false) then state.props.destinationLat else state.props.sourceLat
```

Comment:
in this restructure or code, can we remove the usage of `Just false`? it's good to handle it like Nothing and Just value. This was flagged earlier also. cc: @ashishsingh101